doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -1424,9 +1445,9 @@ Framework.
        logging.info('This is a boring example')
 
 The `logging` module has slightly different log levels than
-Robot Framework. Its levels `DEBUG` and `INFO` are mapped
+Robot Framework. Its levels `DEBUG`, `ERROR`, and `INFO` are mapped
 directly to the matching Robot Framework log levels and `WARNING`
-and everything above is mapped to `WARN`. Custom levels below
+and `CRITICAL` are mapped to `WARN`. Custom levels below
```

Comment:
Is `CRITICAL` really mapped to `WARN`? Isn't it above `ERROR` and thus mapped to `ERROR`?